composer.json

```diff
@@ -19,8 +19,8 @@
     "require": {
         "php": "^8",
         "php-debugbar/php-debugbar": "^2.0",
-        "doctrine/orm": "^2.0|^3.0",
-        "doctrine/dbal": ">=2.3.5 <4.0"
+        "doctrine/orm": "^2.10|^3.0",
+        "doctrine/dbal": ">=3.6"
```

Comment:
can we use ^3.6|^4 ?

Reply:
Yes, we can, 
I only put it that way because Doctrine normally maintains compatibility, and I think that the middleware will be permanent, so there would be no need to modify it if a version 5 comes out.